doc/user_guide/user-guide.md

```diff
@@ -1,11 +1,17 @@
 # Exasol REST API
 
+Exasol REST API is an extension for the Exasol database that provides the ability to interact with the database via REST API endpoints. It is required to use the [Power Apps Connector for Exasol](https://github.com/exasol/power-apps-connector).
```

Comment:
better: "E.g. the REST API is required by the [Power Apps Connector for Exasol]."
Then it is clear that it is not the other way around.

Reply:
See https://github.com/exasol/exasol-rest-api/pull/63